lib/web_ui/lib/src/engine/compositor/image_filter.dart

```diff
@@ -21,4 +21,24 @@ class SkImageFilter implements ui.ImageFilter {
       ],
     );
   }
+
+  final double sigmaX;
+  final double sigmaY;
```

Comment:
Are these expected to be public?

Reply:
Is the engine package considered public in the first place?

I'll make them private anyway.

Actually, I'd have loved to see public getters on the ui class otherwise programmers (and the framework code) don't have any way to tell what kind of ImageFilter they have. We're making == work, and toString prints them out, but there is no programmatic way to get at this information which seems short-sighted, isn't it?

Reply:
The package is not public, but because this class implements a public interface its public members are still visible via `dynamic`, so if there's no strong reason to make something public I'd choose private.

Reply:
Yeah, I agree that this opaqueness is probably overkill. I believe some other classes do this because they immediately encode their data into `TypedData`, making getters more expensive.